pkg/commands/oscommands/cmd_obj.go

```diff
@@ -104,7 +114,21 @@ func (self *CmdObj) GetCmd() *exec.Cmd {
 }
 
 func (self *CmdObj) ToString() string {
-	return self.cmdStr
+	// if a given arg contains a space, we need to wrap it in quotes
+	quotedArgs := lo.Map(self.args, func(arg string, _ int) string {
+		for _, char := range CHARS_REQUIRING_QUOTES {
+			if strings.Contains(arg, string(char)) {
+				return self.quoteFn(arg)
```

Comment:
I never quite understood why we quote with `\"` instead of `"` on Windows, I suppose it has something to do with passing the command to `cmd.exe` or something. But the `ToString` method is only used for logging the command, isn't it? It looks wrong to have the backslashes there, and it makes it harder to copy/paste the command.
<img width="391" alt="image" src="https://github.com/jesseduffield/lazygit/assets/1225667/aac76e76-d88a-41fd-a9dd-5f254d1f57d3">

Reply:
I don't have a ton of Windows experience either. I have a machine at work that I can use to try things out, but I only use it when I have to. 😄 

I was assuming that most Windows users who use git on the command line will be using either GitForWindows' bash command line (the Mingw thing), or WSL. In both cases, you don't want backslashes before the quotes.